frontend/src/component/project/Project/ProjectInsights/ProjectInsights.tsx

```diff
@@ -1,5 +1,12 @@
 import { Box, styled } from '@mui/material';
 import { LeadTimeForChanges } from './LeadTimeForChanges/LeadTimeForChanges';
+import { ProjectHealth } from './ProjectHealth/ProjectHealth';
+
+const Container = styled(Box)(({ theme }) => ({
```

Comment:
I copied this from the lead time component. Every tile shares this basic info

Reply:
Nice